eth/catalyst/simulated_beacon.go

```diff
@@ -261,6 +273,49 @@ func (c *SimulatedBeacon) setCurrentState(headHash, finalizedHash common.Hash) {
 	}
 }
 
+// Commit seals a block on demand.
+func (c *SimulatedBeacon) Commit() common.Hash {
+	withdrawals := c.withdrawals.gatherPending(10)
+	if err := c.sealBlock(withdrawals, uint64(time.Now().Unix())); err != nil {
+		log.Warn("Error performing sealing work", "err", err)
+	}
+	return c.eth.BlockChain().CurrentBlock().Hash()
+}
+
+// Rollback un-sends previously added transactions.
+func (c *SimulatedBeacon) Rollback() {
+	// Flush all transactions from the transaction pools
```

Comment:
Yep you're right, it will only remove the non-local transactions. But since all transactions are send via the RPC, this will not flush any of the transactions

Reply:
Perhaps use  something like this as the eviction? 

`new(big.Int).Mul(big.NewInt(math.MaxInt64), big.NewInt(params.GWei))`